### PR TITLE
Updates of transponder files for camsat XW seris of sats

### PR DESCRIPTION
**Pull request by cureton**
Hi Alex,   A few updates for the transponders for the CAMSAT XW series of satellites

### CODE REVIEW

File: data/trsp/39770.trsp [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Mode U Linear]
```
Comment (csete) [inline data/trsp/39770.trsp]: This seems to be wrong catalog number?

**Comment by csete** [comment]
Thanks for the patch. I'm ready to merge but first please see my comment regarding 39770.trsp

**Comment by cureton** [comment]
Hi Alex,   Will look into it. catalog 39770 is actually sprout but the transponder information is wrong for sprout. I will let you know what I find. 

Is there value in me adding transponder info for  other satellites I can find on the AMSAT web. If so I will make some tools so it is easier for me to cross check the calalog against the name against the source information.

**Comment by csete** [comment]
I think we should only add transponder files on an as needed basis. In the long run, we will interface to  external sources such as Satnogs.

**Comment by cureton** [comment]
Ok, I was not aware of satnogs. Looks interesing

**Comment by csete** [comment]
... and they have a crowd-sourced database:
https://db.satnogs.org/

There is some python script to import data into gpredict, althou I have not yet tried it:
https://github.com/sconklin/satnog-tools

So, we will see how the project evolves.

**Pull request closed without merging**